server/chat-plugins/mafia.ts

```diff
@@ -3988,7 +4018,7 @@ export const commands: Chat.ChatCommands = {
 			let foundTarget = false;
 			for (const entry of ['alignments', 'roles', 'themes', 'IDEAs', 'terms'] as (keyof MafiaData)[]) {
 				const dataEntry = MafiaData[entry];
-				if (from in dataEntry) return this.errorReply(`${from} is already a ${entry.slice(0, -1)}`);
+				if (from in dataEntry) return this.errorReply(`${from} is already a ${entry.toString().slice(0, -1)}`);
```

Comment:
`entry` should already implicitly defined as a string, this extra method call isn't needed.

If its not its likely a problem caused by introducing `[key: string]: any` as a type on MafiaData.